lib/Api256Shim.ts

```diff
@@ -270,6 +270,18 @@ export const addPrivateKeys = (privateKeyA: Uint8Array, privateKeyB: Uint8Array)
  */
 export const subtractPrivateKeys = (privateKeyA: Uint8Array, privateKeyB: Uint8Array): Uint8Array => Recrypt.subtractPrivateKeys(privateKeyA, privateKeyB);
 
+/**
+ * Hashes all possible tri-grams for the given string. The values will be prefixed with the partitionId and salt before being hashed.
+ */
+export const generateHashesForString = (s: string, partitionId: string | undefined, salt: Uint8Array) => Recrypt.generateHashesForString(s, partitionId, salt);
```

Comment:
When we change the order of the parameters on the Rust side like I mentioned in the comment, this can change to 

```suggestion
export const generateHashesForString = (s: string, salt: Uint8Array, partitionId?: string) => Recrypt.generateHashesForString(s, salt, partitionId);
```
Ditto for the other method too.

Reply:
Done

Reply:
Our `schnorrVerify` function should possibly change to this style as well, since that's where I got the undefined syntax from